### PR TITLE
Fixes #1644 icon for individual

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.Core/Model/IndividualFactory.cs [hunk 1]
```diff
@@ -73,8 +73,8 @@ private Individual createStandardIndividual(OriginData originData, Func<IIndivid
          var individual = _objectBaseFactory.Create<Individual>();
          individual.OriginData = originData;
 
-         //default icon for an individual is the name of the species
-         individual.Icon = originData.Species.Name;
+         //default icon for an individual is the icon of the species
+         individual.Icon = originData.Species.Icon;
```
Comment (msevestre) [inline src/PKSim.Core/Model/IndividualFactory.cs]: @Yuri05 I am not sure why this was done like that. Assuming that we did not have an icon property at the beginning

File: src/PKSim.Presentation/Services/BuildingBlockIconRetriever.cs [hunk 2]
```diff
@@ -71,10 +71,10 @@ public ApplicationIcon IconFor(ISimulationComparison simulationComparison)
 
       public ApplicationIcon IconFor(QualificationPlan qualificationPlan) => ApplicationIcons.Formula;
 
-      private string speciesNameFrom(UsedBuildingBlock usedBuildingBlock)
+      private string speciesIconFrom(UsedBuildingBlock usedBuildingBlock)
       {
          var individual = _buildingBlockRepository.ById(usedBuildingBlock.TemplateId) as Individual;
-         return individual == null ? usedBuildingBlock.BuildingBlockType.ToString() : individual.Species.Name;
+         return individual == null ? usedBuildingBlock.BuildingBlockType.ToString() : individual.Species.Icon;
```
Comment (Yuri05) [inline src/PKSim.Presentation/Services/BuildingBlockIconRetriever.cs]: why not just `individual.Icon` ?

Comment (msevestre) [reply src/PKSim.Presentation/Services/BuildingBlockIconRetriever.cs]: fair enough. I did not want to change too much. This should be exactly identical

**Comment by msevestre** [comment]
@Yuri05 Going to merge and will update with your suggestion in next PT

**Pull request merged**